### ISSUE TITLE
100% SSD Usage

### ISSUE THREAD

**Issue by Noctis-** [rocksdanister/lively#163]
**Describe the bug**
I've noticed that my main SSD activity was at 100% for an unusual long time. Having a look at the task manager and the resource monitor didn't show anything that is actively reading or writing to the drive.
At the time, I had 3 browsers open. Edge, Firefox and Chrome, as well as other day to day work related programs (VS2019, slack, outlook etc).

I've figure i'll start to close things to see what triggered the usage. When trying to close chrome tabs, chrome seem to have stop responding. Trying to kill it from the task manager didn't seem to work either.

Eventually i've found that the culprit seems to be the CefSharp.BrowserSubprocess.exe , which is used by lively.
I'm assuming the fact it's using chrome and that hanged might be related (or vice versa).

I was using the "Triangles and Light" at the time, and i have 2 screens.

I'm not sure about how to reproduce though :( 
Hope this might give you an idea or something to think about

![2020-10-16 13_58_15 lively](https://user-images.githubusercontent.com/6160139/96211507-db714f00-0fb7-11eb-8d5e-d41a15bbcdde.png)

**Desktop (please complete the following information):**
 - Windows 10 pro ver 2004
- Which wallpaper was running at the time if any:  "Triangles and Light" 

**Log file**

[logs.zip](https://github.com/rocksdanister/lively/files/5389118/logs.zip)

**Comment by rocksdanister** [comment]
Do you have disk cache enabled in lively settings? (settings - > wallpapers - > web browser)
By default it's set to use memory cache, does this happen with other website wallpapers? 

Also cefsharp process can also be from other applications using the same library.

**Comment by Noctis-** [comment]
Yes, disk cache seems to be selected. 

Re other wallpapers -> i haven't noticed it tbh ... i think i've come across this 100% drive usage maybe 2-3 times in the last months though.
Happy to keep an eye and try to give more details about wallpaper / what was running at the time.

I haven't noticed the cefsharp running with my current set of apps when lively isn't running. I'll keep an eye for that as well.

Impressive work otherwise :) 👍

**Comment by rocksdanister** [comment]
I have disk cache disabled for wallpapers loaded from disk, so it should not matter for offline wallpapers:
https://github.com/rocksdanister/lively/blob/57b1fe6178b5c11cb681e383d3f864d849e43f85/src/livelywpf/livelywpf/Core/Wallpapers/WebProcess.cs#L56

Cef is set to writes its logfile to:
`C:\Users\<UserName>\AppData\Local\Lively Wallpaper\Cef\logfile.txt`
I wonder if that webpage has some uncommented console log message code, the file gets overwritten on startup though.

Are you running an admin install of Lively (C:\\Program files x86).. I should probably change the working directory of cef from its default as write is not allowed there (though it should not matter since I'm setting a custom cache path.)

**Comment by Noctis-** [comment]
CEF logfile.txt is empty.

lively is installed in custom directory in other hdd. how can i tell if it's installed as admin though ? (i've been using them since early 0.6, and usually just installing them over the old ones)

**Comment by rocksdanister** [comment]
damn long time lively fan 😀
Its handled by Inno installer script, I guess one way to check is if you get UAC prompt(if enabled in windows) if you open Lively installer; if prompt is shown then current install is admin mode.

**Comment by Noctis-** [comment]
i've just closed it and ran it again ... no admin mode on the installer (lively_setup_x86_full_v1120.exe)

(sorry...life got in the way :) .

**Comment by rocksdanister** [comment]
I don't think it makes a difference with latest lively, I'll keep this issue open let me know if you find more information.

**Comment by Noctis-** [comment]
Ah ! yes ... this happened earlier ... 

![image](https://user-images.githubusercontent.com/6160139/96678859-e2cc9a00-13b5-11eb-9f8c-64e30ddc1138.png)

so, again ...3 browsers opened ... lively in the background (same wallpaper as above).
i think what happens is that chrome spits the dummy, not lively. 
i've hit a different tab on the chrome and got the "not responding" icon ... 
i took the above screenshot, noting that google crash handler (-> so..i'm assuming chrome)
i've killed lively, chrome is still unresponsive.
trying to kill chrome or the google crash handle failed (no access i believe 🤷 )
restart -> not happening either , my guess, is the process that crashed is locking it
power off, power on ... all good

I'm not 100% sure about the why it happens and if it's actually related to lively anymore though ...if i can figure a way to replicate it consistently, i can try with it on/off ...but again ... i need to find what triggers it

**Comment by andy1128-cpu** [comment]
![Image](https://github.com/user-attachments/assets/6a92169e-b158-419f-8a11-374d2934e0b6) it looks like this when i put other wallpaper on display 2

**Comment by andy1128-cpu** [comment]
![Image](https://github.com/user-attachments/assets/191dc742-18d6-42f2-9554-778be62ca5ee) also saw this on my rain meter @rocksdanister pls help brother.

**Comment by andy1128-cpu** [comment]
it seems ok now ram usage, since i ended task of cefsharp but theres a lot of them i cant tell. wallpaper on display 2 does not work because of it. Not really a tech guy, im just a dude trying to make his pc look good
